python/lsst/meas/base/base.py

```diff
@@ -262,6 +263,23 @@ def fail(self, measRecord, error=None):
                    % self.__class__.__name__)
         raise NotImplementedError(message)
 
+    @staticmethod
+    def getTransformClass():
+        """!
+        Get the measurement transformation appropriate to this plugin.
+
+        This returns subclass of MeasurementTransform, which may be
+        instantiated with details of the algorithm configuration and then
+        called with information about calibration and WCS to convert from raw
+        measurement quantities to calibrated units. Calibrated data is then
+        provided in a separate output table.
+
+        By default, we assume that no transformation is appropriate, and
+        therefore return NullTransform.
+        """
```

Comment:
I have no strong opinion on this, but the current behaviour is what was requested by @TallJimbo; perhaps he'll pitch in here.

Jim -- if it's not obvious from the above, the question is: what should the calibration transform framework do when applied to a measurement plugin which does not have a specific transform specified? I think you previously indicated that it should not copy any of the measurements; the alternative Serge is suggesting is to copy all of them.